exercises/concept/lasagna/lasagna_test.go

```diff
@@ -19,7 +19,7 @@ func TestOvenTime(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			if got := OvenTime; got != tt.expected {
-				t.Errorf("OvenTime(%d)  = %d; want %d", tt.expected, got, tt.expected)
+				t.Errorf("OvenTime(%d)  = %d; want %d", tt.time, got, tt.expected)
```

Comment:
```suggestion
				t.Errorf("OvenTime  = %d; want %d", got, tt.expected)
```

Since `OvenTime` is not a function, It's probably more appropriate to just do this.